commands/python/cluster_rules.py

```diff
@@ -25,34 +18,64 @@ def __getattr__(self, attr):
             "'{}' object has no attribute '{}'".format(self.__class__.__name__, attr)
         )
 
-    def get_certificate_arn(self):
+    def get_certificate_arn(self, for_cluster=False):
```

Comment:
I could probably be persuaded to break this into two functions that call a common internal function to keep the interface a bit more explicit